site/en/blog/crx-scripting-api/index.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Introducing chrome.scripting"
+# description: >
+#   Why and how we implemented color vision deficiency simulation in DevTools and the Blink Renderer.
+layout: "layouts/blog-post.njk"
+authors:
+  - dotproto
+date: 2021-06-07
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/HD1U7BxDVRDcTxNaQJ4C.jpg'
+alt: ''
+tags:
+  - extensions
+---
+
+Manifest V3 introduces a number of changes to Chrome's extension platform. In this post, we'll be
+exploring the motivations and changes introduced by one of the more notable changes: the
+introduction of the `chrome.scripting` API.
+
+## What is chrome.scripting?
+
+As the name might suggest, `chrome.scripting` is a new namespace introduced in Manifest V3
+responsible for script and style injection capabilities.
+
+Developers that have created Chrome extensions in the past may be familiar with Manifest V2 methods
+on the Tabs API like `chrome.tabs.executeScript` and `chrome.tabs.insertCSS`. These methods allow
```

Comment:
Should we link to the tabs API methods?

Reply:
Definitely. I meant to do this before but it slipped my mind. Thanks for calling this out.